### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Download VSCode_Keyboard: "62 bit and 32 bit" links are not accessible by keyboard

### ISSUE THREAD

**Issue by Arjun553** [microsoft/vscode-docs#2659]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
**Environment Details:**
code.visualstudio.com/download
Browser: Edge, Chrome

**Repro Steps:**
1)Hit the URL:code.visualstudio.com/download
2)Verify whether all the controls and links are accessible by using keyboard or not

**Actual Result:**
 "62 bit and 32 bit" links are not accessible by keyboard

**Expected Result:**
 All  links present in the page should be  accessible by keyboard

**User Impact:**
Keyboard users will face difficulty in understanding and accessing the controls if   "62 bit and 32 bit" links are not accessible by keyboard

**Mas reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by Arjun553** [comment]
![Bug_2659_Keyboard](https://user-images.githubusercontent.com/50437579/57452741-33b00780-7282-11e9-9d60-d05dad00c6fc.png)

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
